modules/serverless-logic-install-web-console.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * /serverless/install/install-serverless-operator.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-install-web-console_{context}"]
+= Installing the {ServerlessLogicOperatorName} from the web console
+
+You can install the {ServerlessLogicOperatorName} from the OperatorHub by using the {ocp-product-title} web console. Installing this Operator enables you to install and use serverless workflows.
+
+.Prerequisites
+
+* You have cluster administrator permissions on {ocp-product-title}, or you have cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
+* For {ocp-product-title}, your cluster has the Marketplace capability enabled or the Red Hat Operator catalog source configured manually.
+
+
+* You have logged in to the  web console.
+
+.Procedure
+
+. In the web console, navigate to the *Operators* -> *OperatorHub* page.
+
+. Scroll, or type the keyword *Serverless Logic* into the *Filter by keyword* box to find the {ServerlessLogicOperatorName}.
+
+. Review the information about the Operator and click *Install*.
+
+. On the *Install Operator* page:
+
+.. The *Installation Mode* is *All namespaces on the cluster (default)*. This mode installs the Operator in the default `openshift-serverless-logic` namespace to watch and be made available to all namespaces in the cluster.
+
+.. The *Installed Namespace* is `openshift-serverless-logic`.
+
+.. Select an *Update Channel*.
++
+* The *stable* channel enables installation of the latest stable release of the {ServerlessLogicOperatorName}. The *stable* channel is the default.
+
+.. Select the *stable* channel as the *Update Channel*. The *stable* channel will enable installation of the latest stable release of the {ServerlessLogicOperatorName}.
```

Comment:
Is this step repeated? I see the same format in the [Serverless Operator](https://docs.openshift.com/serverless/1.33/install/install-serverless-operator.html#serverless-install-web-console_install-serverless-operator) instructions.

```suggestion
.. Select an *Update Channel*.
+
.. Select the *stable* channel as the *Update Channel*. The *stable* channel will enable installation of the latest stable release of the {ServerlessLogicOperatorName}.
```